File: jscomp/test/nested_obj_literal.js
'use strict';


var structural_obj = do
  x: do
    y: do
      z: 3
    end
  end
end;

var f_record = do
  x: do
    y: do
      z: 3
    end
  end
end;

exports.structural_obj = structural_obj;
exports.f_record = f_record;
--[ No side effect ]--

File: jscomp/test/nested_obj_literal.lua
__console = {log = print};


structural_obj = {
  x = {
    y = {
      z = 3
    }
  }
};

f_record = {
  x = {
    y = {
      z = 3
    }
  }
};

exports = {};
exports.structural_obj = structural_obj;
exports.f_record = f_record;
return exports;
--[[ No side effect ]]
